### DISCUSSION TITLE
Book note - any way to make List default view (instead of Grid)?

### DISCUSSION THREAD

**Discussion by contributor** [TriliumNext/Trilium#3561] [General]
Hi,

Every time I create or change a note to Book - I have to change from Grid to Text. My notes are all text. Personally, I never find any reason to use Grid view type at all. It may be good for images, but I don't store standalone images in Trilum

**Comment by zadam** [comment]
Add a `#viewType(inheritable)=list` to the root note.

**Comment by contributor** [comment]
Thank so much. Your app is amazing!
